packages/DraftEditor/src/index.tsx

```diff
@@ -128,6 +129,16 @@ const DraftEditor: FunctionComponent<IDraftEditorProps> = ({
     EditorState.createEmpty()
   );
 
+  useEffect(() => {
+    if (value) {
+      const rawObjectValue = JSON.parse(value);
+      const state = EditorState.createWithContent(
```

Comment:
Please :type the state, to ensure you perform a correct format conversion.

Reply:
Will do! 👍